### ISSUE TITLE
Accessibility of oTree content

### ISSUE THREAD

**Issue by nfh102** [oTree-org/oTree#118]
Most experiments designed with oTree seem to be lacking key accessibility considerations, such as field and button labeling, proper use of headers, alternative text for images, proper table coding, etc. (examples from [https://otree-more-demos.herokuapp.com/demo](https://otree-more-demos.herokuapp.com/demo)). I tried searching for accessibility information in the product documentation, and found nothing. Can support for accessibility be a consideration for future releases? I'm happy to work with the developer community with guidance, acceptance testing, etc.
